packages/manager/modules/pci/src/projects/project/kubernetes/details/restrictions/IP.class.js

```diff
@@ -0,0 +1,12 @@
+export default class IP {
```

Comment:
**nitpick**:
Regarding https://github.com/ovh/manager/pull/4316/files#diff-182ca3baad5b54648cbcbcd1b8ebdb575febbba4a6bdbc25f165f8fe9d2f09e3R75, should the name of the class be `IPv4` ?